chore: Update firewall info for enrichers that access external endpoints (#415)
CluedIn-io/CluedIn.Documentation: docs/020-deployment/howtos/100-configure-firewall.md

| `mcr.microsoft.com` | 443 | Microsoft container registry |
| `acme-v02.api.letsencrypt.org` | 443 | (Optional) Let's Encrypt service. Only required if not supplying own certificate |

### Enricher Examples

> Below are optional additions to the above and are only required if you enrichers. Below are two of our common enrichers, but each enricher will have their own endpoints configured. If you require assistance with what endpoint is used for each CluedIn enricher, please reach out to CluedIn support who will be happy to assist.

Reviewer: ```suggestion
Below are optional additions to the above and are only required if you use enrichers. Below are two of our common enrichers, but each enricher will have its own endpoint configured. If you require assistance with what endpoint is used for each CluedIn enricher, please reach out to CluedIn support who will be happy to assist.
```
